package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "csharp",
   "publisher": "ms-vscode",
-  "version": "1.13.1",
+  "version": "1.14.1",
```

Comment:
This should be `1.14.0-beta1`

Reply:
@gregg-miskelly Since we uploaded the vsdbg as 1.14.1, will making this 1.14.0-beta2 be a problem?